inventory/group_vars/ca.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+# See https://github.com/openfoodfoundation/ofn-install/wiki/Setup for more info
+checkout_zone: Canada
+country_code: CA
+currency: CAD
+locale: en
+available_locales: en,es    # Available locales for OFN instance
```

Comment:
wait. Is `es` right here? shouldn't it be `fr_CA` instead?

Reply:
Well spotted! It was originally provisioned with es as an example but got changed to fr_CA. I added another commit with the missing timezone as well.